### PR TITLE
Updated schema for new hacker eval

### PR DESCRIPTION
**Pull request by andykwan1124**
## Description
Updated schema based on the hacker eval changes.

Changes I made were based on the behavior shown in [https://www.figma.com/file/UEDqyTnpzM3z95ODhrG2u0/Hacker-Application-Evaluator?node-id=551%3A1005](url).

Plz keep in mind the schema is always subject to change. Feel free to reach out or even make a PR to update the schema as you see fit. 

With the way I designed the schema, a new collection storing all of the nwplus members will need to be created in the database. This can be most likely be done with firebase cloud functions. 

Short explanation on the changes:

- An `admin` collection will be created to store some info for all current team members.
  * This is for storing the evaluations that are assigned to each member (for now)
  * If we develop more internal tools later on and want to store info for each member, this collection should be able to handle that
- Each admin will be associated with a `HackathonEvaluations` collection
  * Each document in that collection will represent the evaluations assigned to that admin for a certain hackathon
  * Each document in that collection will contain: 
    * Number of items assigned to that admin
    * Number of items completed by that admin
    * A list of `SectionsAssigned` objects
- Each `SectionsAssigned` object will contain:
  * An array of `Range` objects representing the `numericalIds` of applications
  * The section of the applications included in the `Range` objects that are assigned to the admin (github, resume...)
- Each `Application` object is also updated with a new field for `numericalIds` and new `score` fields

### CODE REVIEW

**Comment by andykwan1124** [comment]
Will be tagging everyone for review since we should all know how the schema will look like.

File: nw-firestore-schema.ts [hunk 1]
```diff
@@ -63,6 +63,33 @@ interface DayOfEvent {
   description: string;
 }
 
+type Team = "Dev" | "Design" | "Marketing" | "Content Writing" | "cmd-f" | "Hackcamp" | "nwHacks" | "Engagement" | "Sponsorship"
```
Comment (anneguo3) [inline nw-firestore-schema.ts]: Can we add "Pres" as a team too? Also will it just be an individual has a Team, and Sections Assigned to them?

Comment (andykwan1124) [reply nw-firestore-schema.ts]: Sure! I have updated the description of the PR with an explanation for the changes, but yes that's the main idea.

**Review by kevin-zou — APPROVED** [review]
🚀 LFG

**Pull request merged**